docs/presentations/pycon2024/maintainers.qmd

```diff
@@ -0,0 +1,322 @@
+---
+title: "Test 20 databases on every commit"
+subtitle: "You can, it's not hyperbole"
+author:
+  - Phillip Cloud
+execute:
+  echo: true
+format:
+  revealjs:
+    footer: <https://ibis-project.org>
+    chalkboard: true
+    # https://quarto.org/docs/presentations/revealjs/themes.html#using-themes
+    theme: dark
+---
+
+# what
+
+## maybe this is you
+
+![](./docker-eye-roll.gif){fig-align="center"}
+
+## or this
+
+![](./wonka.png){fig-align="center"}
+
+## or maybe even this
+
+![](./basement-ci.jpeg){fig-align="center"}
+
+## this talk isn't earth shattering
+
+opinions informed by experience
+
+:::: {.columns}
+
+::: {.column width="50%"}
+### overview
+
+- what we learned about maintenance building Ibis
+- day to day of supporting 20+ databases
+- unique challenges
+:::
+
+::: {.column width="50%"}
+### topics
+
+- some docker stuff
+- some packaging stuff
+- pytest plugins you should be using
+:::
+::::
+
```

Comment:
this slide is a lot of text -- perhaps transition in one of the columns? but it might be fine

Reply:
removed a bit of it